### PR TITLE
Fixed momentjs deprecation warning. Closes #47.

### LINKED ISSUE #47: moment.js deprecation message appearing in logs.
Log's are displaying a deprecation message from [moment.js](https://github.com/moment/moment):

> Deprecation warning: moment construction falls back to js Date. This is discouraged and will be removed in upcoming major release. Please refer to https://github.com/moment/moment/issues/1407 for more info.

### PR DESCRIPTION
**Pull request by arb**
Changes `moment` construction to avoid moment deprecation warning.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ internals.GoodConsole.prototype.init = function (stream, emitter, callback) {
 
 internals.GoodConsole.prototype._printEvent = function (event) {
 
-    var m = Moment.utc(event.timestamp);
+    var m = Moment(new Date(parseInt(event.timestamp, 10)));
```
Comment (geek) [inline lib/index.js]: why not create like:

```
var m = Moment(event.timestamp);
```

Comment (arb) [inline lib/index.js]: Incase `event.timestamp` is a string. Then you'll get the deprecation warning which is what this is supposed to fix.

Comment (geek) [inline lib/index.js]: I see that now... but you don't need to make it a Date

```
var m = Moment(parseInt(event.timestamp, 10));
```

Comment (arb) [inline lib/index.js]: You got me! PR updated.

**Pull request merged**